### ISSUE TITLE
Can't save search providers

### ISSUE THREAD

**Issue by RicardoCino** [SickRage/sickrage-issues#786]
@OmgImAlexis 
I'm having an issue while changing the settings of the Search Providers, when clicking on a checkbox in the search providers list it gives me this error in the developer console:

`Uncaught TypeError: $(...).refreshProviderList is not a function`

Also, when I'm on that page it gives the following message in the console wich might be interesting in this situation:

`This function need to be filled with ConfigProviders.js but can't be as we've got scope issues currently.`

I installed SickRage on an Synology NAS ( ds216play ), the installation is completely clean. While installing I used sickbeard-custom and checked out the master branch of sickrage. I didn't do anything special with the installation as it's running know everything is untouched.

If you need more information, let me know.

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by fernandog** [comment]
@RicardoCino please send a print of localhost:8081/config or whatever address/port

**Comment by RicardoCino** [comment]
Here is the screenshot.
![sickrage sickrage configuration](https://cloud.githubusercontent.com/assets/2735602/12867407/05910c0c-cced-11e5-9f76-0a50dd1a4087.png)

**Comment by fernandog** [comment]
We will have to wait @OmgImAlexis

**Comment by RicardoCino** [comment]
I have the idea it's because there used to be an original sickbeard installation on the Synology NAS. Since I have the time I'm going to backup all the files and reinstall the nas it self. If this works I probably let it know tonight / tomorrow ( data backup is gonna take a while ).

**Comment by OmgImAlexis** [comment]
I have a feeling this is because of something someone moved, probably @miigotu 

@SickRage/collaborators anyone remember who moved the js functions around? The function is  probably out of scope now.

**Comment by OmgImAlexis** [comment]
Actually since that function should be in `configProviders.js` there shouldn't be an issue.

**Comment by RicardoCino** [comment]
As far as I could see the function is there. So after a complete re-install of my Nas ( to be sure it's clean ) and installation of SickRage I'm still getting the same error message. Even when I'm trying to use the original sickbeard. Can't seem to get either one of them to work right now.

I'm in full doubt if this is because of the code or the system.

**Comment by fernandog** [comment]
Try clean browser cache

**Comment by RicardoCino** [comment]
I've done that a couple times withouth any succes. Keeping in mind that it's probably a browser issue I tried a different browser and it seems to be working. After that I cleaned my browser cache twice (chrome) again with no succes, after completely removing google chrome and reinstalling it it's finally working. This is probably something that should be fixed in the future because I don't think it's really a good solution this way.


Anyway I thank you all for the help, and as far as I'm concerned this issue can be closed.
Goodnight!
